crypto_sign/dilithium2/clean/packing.c

```diff
@@ -4,188 +4,186 @@
-*              - const unsigned char key[]: output byte array for key
-*              - const unsigned char tr[]: output byte array for tr
+* Arguments:   - const uint8_t rho[]: output byte array for rho
+*              - const uint8_t key[]: output byte array for key
+*              - const uint8_t tr[]: output byte array for tr
 *              - const polyvecl *s1: pointer to output vector s1
 *              - const polyveck *s2: pointer to output vector s2
 *              - const polyveck *r0: pointer to output vector t0
-*              - unsigned char sk[]: byte array containing bit-packed sk
+*              - uint8_t sk[]: byte array containing bit-packed sk
 **************************************************/
-void PQCLEAN_DILITHIUM2_CLEAN_unpack_sk(unsigned char rho[SEEDBYTES],
-                                        unsigned char key[SEEDBYTES],
-                                        unsigned char tr[CRHBYTES],
-                                        polyvecl *s1,
-                                        polyveck *s2,
-                                        polyveck *t0,
-                                        const unsigned char sk[CRYPTO_SECRETKEYBYTES]) {
-    unsigned int i;
-
-    for (i = 0; i < SEEDBYTES; ++i) {
+void PQCLEAN_DILITHIUM2_CLEAN_unpack_sk(
+    uint8_t *rho,
+    uint8_t *key,
+    uint8_t *tr,
+    polyvecl *s1,
+    polyveck *s2,
+    polyveck *t0,
+    const uint8_t *sk) {
+    for (size_t i = 0; i < SEEDBYTES; ++i) {
         rho[i] = sk[i];
     }
     sk += SEEDBYTES;
 
-    for (i = 0; i < SEEDBYTES; ++i) {
+    for (size_t i = 0; i < SEEDBYTES; ++i) {
         key[i] = sk[i];
     }
     sk += SEEDBYTES;
 
-    for (i = 0; i < CRHBYTES; ++i) {
+    for (size_t i = 0; i < CRHBYTES; ++i) {
         tr[i] = sk[i];
     }
     sk += CRHBYTES;
 
-    for (i = 0; i < L; ++i) {
+    for (size_t i = 0; i < L; ++i) {
         PQCLEAN_DILITHIUM2_CLEAN_polyeta_unpack(&s1->vec[i], sk + i * POLETA_SIZE_PACKED);
     }
     sk += L * POLETA_SIZE_PACKED;
 
-    for (i = 0; i < K; ++i) {
+    for (size_t i = 0; i < K; ++i) {
         PQCLEAN_DILITHIUM2_CLEAN_polyeta_unpack(&s2->vec[i], sk + i * POLETA_SIZE_PACKED);
     }
     sk += K * POLETA_SIZE_PACKED;
 
-    for (i = 0; i < K; ++i) {
+    for (size_t i = 0; i < K; ++i) {
         PQCLEAN_DILITHIUM2_CLEAN_polyt0_unpack(&t0->vec[i], sk + i * POLT0_SIZE_PACKED);
     }
 }
 
 /*************************************************
-* Name:        pack_sig
+* Name:        PQCLEAN_DILITHIUM2_CLEAN_pack_sig
 *
 * Description: Bit-pack signature sig = (z, h, c).
 *
-* Arguments:   - unsigned char sig[]: output byte array
+* Arguments:   - uint8_t sig[]: output byte array
 *              - const polyvecl *z: pointer to vector z
 *              - const polyveck *h: pointer to hint vector h
-*              - const poly *c: pointer to challenge polynomial
+*              - const poly *c: pointer to PQCLEAN_DILITHIUM2_CLEAN_challenge polynomial
```

Comment:
?

Reply:
Probably got renamed by a refactoring operation (there's a function called `challenge` in `sign.c`).